app/views/foods/index.html.erb

```diff
@@ -0,0 +1,34 @@
+<%= render "shared/header" %>
+  <section class="container">
+    <div class="">
+      <%= link_to 'foods/new', :remote => true do %>
+        <input type="button" class="btn btn-primary mb-3" value="Add Food">
```

Comment:
`Add Food` button should be on the top right of the food table on in `foods index` page as it can be seen in [this](https://raw.githubusercontent.com/microverseinc/curriculum-rails/main/recipe-app/images/food_index.png?token=GHSAT0AAAAAABV2SE5E3UQPM6MRQFNQFN2OYVWFKNA) wireframe.

![add_btn_foods](https://user-images.githubusercontent.com/35504149/175662313-7fd99276-243c-47e8-bcc0-67837350fe6a.png)

Reply:
Done with this styling err...
![CaptureXXII](https://user-images.githubusercontent.com/83566118/175697216-504691a2-e618-403c-8f66-cd1a5bf68060.PNG)

Reply:
:+1: Fixed :heavy_check_mark: